circle.yml

```diff
@@ -10,13 +10,8 @@ dependencies:
     - make install
 test:
   override:
-    - make lint
-    - make test-all
-    - behave --junit --junit-directory $CIRCLE_TEST_REPORTS/behave/junit.xml:
+    - case $CIRCLE_NODE_INDEX in 0) make lint; make test-all; make docs ;; 1) behave integration-tests/ --junit --junit-directory $CIRCLE_TEST_REPORTS/behave/junit.xml ;; esac:
```

Comment:
Essentially running parallel just hammers the API and causes lots of dropped connections - so lots of tests fail. I tried putting retry function around the boto calls, which works, but I can't wrap the boto calls Sceptre makes. So when Sceptre experiences a dropped connection - it fails. I'm sure there is a way, but might save it for another PR.

The line above essential runs the lint, unit-tests, docs in a separate container to integration tests - which save a little bit of time.